nav2_msgs/srv/SaveMap3D.srv

```diff
@@ -0,0 +1,12 @@
+# URL of map resource
```

Comment:
line6-8:- 
```
geometry_msgs/Vector3 origin
geometry_msgs/Quaternion orientation
bool as_binary
```

Reply:
Do you need that though?

Reply:
origin and orientation can be squashed into one ```origin``` but ```as_binary``` will be needed otherwise we won't be able to give flexibility b/w binary and ASCII formats.

Reply:
Why do you need to set an origin if the pointcloud is already aligned and data is being processed relative to some map frame? All the data is at real-world coordinates like (5,4,2), relative to the identity origin. Why add more complexity to it?

Reply:
Then it's ok, some previous conversation has mentioned that origin/orientation is important so I designed it in the same way. One thing to be noted:- the yaml saved by ```map_saver``` for 2D also saves origin[x, y, yaw]

Reply:
https://github.com/ros-planning/navigation2/pull/1898#discussion_r470508183

Reply:
I am also with merging them as `origin`. Previously this was in the merged condition but with a different name `view-point`  after some objections instead of renaming this, I split it to `origin` and `orientation`.